pages.fr/common/avrdude.md

```diff
@@ -0,0 +1,20 @@
+# avrdude
+
+> Pilotes pour programmer les microcontrôleurs Atmel AVR.
+> Plus d'informations : <https://www.nongnu.org/avrdude/>.
+
+- Lire le contenu du microcontrôleur :
```

Comment:
Hi ! thank's for the review ! 

Well, technically your sentence is correct, but that's a literal translation.
I'll explain why: 
When we translate `Read` in french 90% of the time the correct answer is `Lire`
However, `Lire` can only be used on something that's readable for humans.

For example:

- Nicolas lis un livre (Nicolas reads a book)
- Nicolas lis l'étiquette d'une boite à chaussure. (Nicolas reads the label of a shoe box )

As you can see for the second exemple i can't say (Nicolas reads a shoe box), i need to specify what he reads because read a shoe box is not correct.


Back to our case, `Read AVR microcontroller` in english i clearly understand that we are talking about the program inside the microcontroller. 
However in french if i translate literally it would means that reading something that's wrote on the microcontroller. (like the serial number, the model, etc ...) not the content of it.

That's why i added `the content` translated `le contenu` in my sentence to avoid confusion.

I hope that the explanation is clear ^^"  

PS : yes french is weird sometimes ....

Reply:
Wow, Thanks for the information. I think we can have your version itself.